miio/integrations/roidmi/vacuum/roidmivacuum_miot.py

```diff
@@ -558,7 +558,6 @@ def status(self) -> RoidmiVacuumStatus:
         return RoidmiVacuumStatus(
             {
                 prop["did"]: prop["value"] if prop["code"] == 0 else None
-                # max_properties limmit to 10 to avoid "Checksum error" messages from the device.
```

Comment:
Also, this comment should be removed, since the `max_properties` is not properly limited at all, and there is also a typo in this comment. Maybe it is copy-pasted from somewhere else.

Reply:
Yeah, that's probably just due to some copy&pasting. Please remove changes to roidmivacuum_miot from this PR (as they are unrelated, but feel free to create a separate PR to remove those if you wish!) and then this is good to go :+1: